homeassistant/components/tado/climate.py

```diff
@@ -203,15 +200,14 @@ def __init__(
         supported_hvac_modes,
         supported_fan_modes,
         support_flags,
-        device_info,
     ):
         """Initialize of Tado climate entity."""
         self._tado = tado
-        super().__init__(zone_name, device_info, tado.device_id, zone_id)
+        super().__init__(zone_name, tado.home_id, zone_id)
 
         self.zone_id = zone_id
         self.zone_type = zone_type
-        self._unique_id = f"{zone_type} {zone_id} {tado.device_id}"
+        self._unique_id = f"{zone_type} {zone_id} {tado.home_id}"
```

Comment:
Does the unique id change?

Reply:
Nope, it doesn't.
BTW, I didn't change this, I just renamed the variable from `device_id` to `home_id`, which is the real ID used by the current home-assistant code.

Reply:
Ok, good. Please do this change in a separate PR. We want to separate clean ups from feature changes.

Reply:
Ok, I will add a separate PR for that once this PR is merged.